src/main/kotlin/org/pkl/intellij/util/Util.kt

```diff
@@ -233,3 +234,5 @@ fun parseUriOrNull(uriStr: String): URI? =
   } catch (_: URISyntaxException) {
     null
   }
+
+fun Path.dropRoot(): Path = toAbsolutePath().subpath(0, count())
```

Comment:
You can't `count()` until after you've taken the absolute path, right?

```suggestion
fun Path.dropRoot(): Path = toAbsolutePath().let { it.subpath(0, it.count()) }
```

Reply:
Indeed! I changed this function to also call `toAbsolutePath` and forgot to update `count`...